### ISSUE TITLE
24973364: UITableView is missing accesibilityIdentifier field in Interface Builder

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#7714]
#### Description

Summary:
All accessibility properties are missing for UITableViews in Interface Builder. At least accessibilityIdentifier is crucial for some automated tests.

Steps to Reproduce:
Add a UITableView to a view in Interface Builder. Switch to Indentity Inspector -> no accessibilty properties

Expected Results:
all or at least some accessibilty properties

Actual Results:
no accessibilty properties

Regression:

Notes:
## 

Product Version: Xcode 7.2.1 (7C1002)
Created: 2016-03-04T10:47:54.851140
Originated: 2016-03-04T11:47:00
Open Radar Link: http://www.openradar.me/24973364
